kodilla-good-patterns/src/main/java/com/kodilla/good/patterns/challenges/thirdChallenge/InformationOfAirports.java

```diff
@@ -0,0 +1,31 @@
+package com.kodilla.good.patterns.challenges.thirdChallenge;
+
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public class InformationOfAirports {
+
+   public void printListOfStartAirports(Map<Integer, Flight> mapOfAllFlights) {
```

Comment:
te metody mogą być statyczne, to wtedy nie trzeba tworzyć obiektu klasy InformationOfAirports